move-to-infra/move-to-infra.sh

```diff
@@ -0,0 +1,179 @@
+  do
+    PODS_READY=$(oc get pod  -l "${1}" --field-selector=status.phase==Running -o jsonpath="{.items[*].metadata.name}" -n${NAMESPACE} | wc -w)
+    if [[ $PODS_READY -eq $EXPECTED_PODS ]]; then
+      echo "...found expected pods running"
+      break
+    fi
+    ATTEMPTS=$[$ATTEMPTS+1]
+    echo "Sleep 10s and check again."
+    sleep 10
+  done
+  if [[ $PODS_READY -ne $EXPECTED_PODS ]]; then
+    echo "...not found expected pods. exit"
+    exit 1
+  fi
+}
+
+# create config map to move 
+monitoringConfigMap()
+{
+  cat <<EOT > cluster-monitoring-configmap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |+
+    alertmanagerMain:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: infra
+        value: reserved
+        effect: NoSchedule
+      - key: infra
+        value: reserved
+        effect: NoExecute
+    prometheusK8s:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: infra
+        value: reserved
+        effect: NoSchedule
+      - key: infra
+        value: reserved
+        effect: NoExecute
+    prometheusOperator:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: infra
+        value: reserved
+        effect: NoSchedule
+      - key: infra
+        value: reserved
+        effect: NoExecute
+    grafana:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: infra
+        value: reserved
+        effect: NoSchedule
+      - key: infra
+        value: reserved
+        effect: NoExecute
+    k8sPrometheusAdapter:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: infra
+        value: reserved
+        effect: NoSchedule
+      - key: infra
+        value: reserved
+        effect: NoExecute
+    kubeStateMetrics:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: infra
+        value: reserved
+        effect: NoSchedule
+      - key: infra
+        value: reserved
+        effect: NoExecute
+    telemeterClient:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: infra
+        value: reserved
+        effect: NoSchedule
+      - key: infra
+        value: reserved
+        effect: NoExecute
+    openshiftStateMetrics:
+      nodeSelector:
+        node-role.kubernetes.io/infra: ""
+      tolerations:
+      - key: infra
+        value: reserved
+        effect: NoSchedule
+      - key: infra
+        value: reserved
+        effect: NoExecute
+EOT
+}
+
+# exit in case of any errors
+set -e
+
+echo "Moving apps into infra nodes..."
+
+# Check if infra nodes are available
+INFRA_NODES=$(oc get nodes -l node-role.kubernetes.io/infra -o jsonpath='{.items[*].metadata.name}' | wc -w)
```

Comment:
I don't see the 'oc login' in the script - is it assumed that oc client session already connected to ARO when this script is executed?

Reply:
It was not required. I use docker image: registry.redhat.io/openshift4/ose-cli
PR: https://github.com/finastra-engineering/fusion-operate-charts/pull/8